content/_releases/22/50.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Product Update - v22.50
+description: Details of the product update v22.50 from 15th of December 2022.
+layout: rel
+version: "22.50"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-12-15
+redirect_from:
+  - /releases/22.50.html
+---
+
+## Features
+
+### Recipes list: Use Interface
+
+In this release we introduce a new dedicated page for recipe management accessible
+through new Recipes menu item available under Integrate section. Now all recipe
```

Comment:
the new 'Recipes'....the 'Integrate' section

Reply:
This must stay capital letter since this is the name of the section on our UI.